doc/usage.md

```diff
@@ -1,6 +1,6 @@
-# Usage
```

Comment:
@stefanv Renamed this so that our users don't try to clone and run F on their laptops